addons/website/views/new_page_template_templates.xml

```diff
@@ -1,6 +1,176 @@
+                </div>
+            </div>
+        </div>
+        <div data-name="Field" class="s_website_form_field mb-3 col-12 s_website_form_model_required" data-type="email">
+            <div class="row s_col_no_resize s_col_no_bgcolor">
+                <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="oub62hlfgjwf">
+                    <span class="s_website_form_label_content">Your Email</span>
+                    <span class="s_website_form_mark"> *</span>
+                </label>
+                <div class="col-sm">
+                    <input class="form-control s_website_form_input" type="email" name="email_from" required="" data-fill-with="email" id="oub62hlfgjwf"/>
+                </div>
+            </div>
+        </div>
+        <div data-name="Field" class="s_website_form_field mb-3 col-12 s_website_form_custom" data-type="char">
+            <div class="row s_col_no_resize s_col_no_bgcolor">
+                <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="o291di1too2s">
+                    <span class="s_website_form_label_content">Your Company</span>
+                </label>
+                <div class="col-sm">
+                    <input class="form-control s_website_form_input" type="text" name="company" data-fill-with="commercial_company_name" id="o291di1too2s"/>
+                </div>
+            </div>
+        </div>
+        <div data-name="Field" class="s_website_form_field mb-3 col-12 s_website_form_model_required" data-type="char">
+            <div class="row s_col_no_resize s_col_no_bgcolor">
+                <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="oqsf4m51acj">
+                    <span class="s_website_form_label_content">Subject</span>
+                    <span class="s_website_form_mark"> *</span>
+                </label>
+                <div class="col-sm">
+                    <input class="form-control s_website_form_input" type="text" name="subject" required="" id="oqsf4m51acj"/>
+                </div>
+            </div>
+        </div>
+        <div data-name="Field" class="s_website_form_field mb-3 col-12 s_website_form_custom s_website_form_required" data-type="text">
+            <div class="row s_col_no_resize s_col_no_bgcolor">
+                <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="oyeqnysxh10b">
+                    <span class="s_website_form_label_content">Your Question</span>
+                    <span class="s_website_form_mark"> *</span>
+                </label>
+                <div class="col-sm">
+                    <textarea class="form-control s_website_form_input" name="description" required="1" id="oyeqnysxh10b" rows="3">
+                    </textarea>
+                </div>
+            </div>
+        </div>
+    </xpath>
+</template>
```

Comment:
Is this a kind of copy paste from the contact us page definition (in website/data.xml)?

Let's create a commit on top of this PR which encapsulate the contactus form in a template and the contactus defiition in data.xml should do a t-call of it.
And add t-ignore on it so we can still full edit it.

And then here, just t-call this template.

If not clear we can discuss it :+1:

Reply:
Somehow when doing this, blocks cannot be dropped anymore in the "Contact Us" page.
It seems that instead of being put on the `wrap`, the branding gets put on the `span` that precedes it.

Keeping this hard-coded form content for the time being.